.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+- 8
```

Comment:
Think we should build against latest stable and latest LTS https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions